src/Viz.js

```diff
@@ -660,6 +674,16 @@ If *data* is not specified, this method returns the current primary data array,
     return arguments.length ? (this._depth = _, this) : this._depth;
   }
 
+  /**
+      @memberof Viz
+      @desc If *value* is specified, sets the description accessor to the specified string and returns the current class instance.
+      @param {String} [*value*]
+      @chainable
+  */
+  desc(_) {
+    return arguments.length ? (this._svgDesc =  _ ) : this._svgDesc;
```

Comment:
This line is not correct, and causes an error when trying to use the method. You are assigning the `_svgDesc`, but not returning anything (so it's not chainable). Referencing the method above, it should be this:
```js
return arguments.length ? (this._svgDesc =  _, this) : this._svgDesc;
```

Please make sure that you have a javascript linter installed in your editor. That was the first thing to tip me off that this is where there are errors (`npm test` also fails).

What my Atom.io linting looks like:

<img width="609" alt="screen shot 2018-06-18 at 11 40 22 am" src="https://user-images.githubusercontent.com/1847581/41546771-8149a384-72ec-11e8-857c-fe89c13133ed.png">